### ISSUE TITLE
S3 Content-Type header set incorrectly.

### ISSUE THREAD

**Issue by JT-Bruch** [milliHQ/terraform-aws-next-js#277]
When our build is generated, there is a folder structure that looks like as follows:

However, the issue is that in the S3 bucket, the upload is setting the content-type header as `application/emcascript` on only the `es` page itself which is causing the page to be rendered as text instead of a website.

![image](https://user-images.githubusercontent.com/7430562/151893538-3a35f9c0-1a9c-45da-a956-e1c7cd662246.png)

![image](https://user-images.githubusercontent.com/7430562/151893558-e29277e2-8004-4e43-b7d6-3639d85a0b65.png)

this is what our next.config.js looks like

```
const withPlugins = require('next-compose-plugins');

module.exports = withPlugins([], {
  experimental: {
    externalDir: true,
  },
  i18n: {
    locales: ['en', 'es'],
    defaultLocale: 'en',
    localeDetection: false,
  },
  images: {
    domains: ['res.cloudinary.com'],
  },
  publicRuntimeConfig: {
    IS_DEV: process.argv.includes('dev'),
  },
  async headers() {
    return [
      {
        source: '/es',
        headers: [
          {
            key: 'Content-Type',
            value: 'text/html; charset=utf-8',
          },
        ],
      },
    ]
  },
  async rewrites() {
    return [
      {
        source: '/es',
        destination: '/es.html',
      },
    ]
  },
```

Solution:

I can manually go in and set the header to text/plain and then the page renders properly.  Any ideas on how I can fix this?

Thanks!

**Comment by JT-Bruch** [comment]
If I can't get this to work, I am going to have to move our production from AWS -> Vercel, and I would rather not do that.

**Comment by ofhouse** [comment]
Hi, it looks like the [`mime-types`](https://www.npmjs.com/package/mime-types) package, we use internally for determining the content-type header, thinks that `es` is the file ending:

https://github.com/milliHQ/terraform-aws-next-js/blob/main/packages/deploy-trigger/src/deploy-trigger.ts#L88-L90

**Comment by ofhouse** [comment]
This has now been fixed in [`v0.11.4`](https://github.com/milliHQ/terraform-aws-next-js/releases/tag/v0.11.4), thanks for reporting!
